<launch>
    
    
    <arg name="target" default="person" />
    <arg name="focal_length" default="747.0" />
    <arg name="known_target_width_cm" default="50.0" />
    <arg name="min_distance" default="0.3" /> 
    <arg name="max_distance" default="10.0" /> 

    <arg name="speed" default="0.3" /> 

    <arg name="angular_scale_factor" default="1.0" />  


    <arg name="box_percentage_from_image_to_stop" default="0.8" />

    <arg name="enable_gestures" default="false" />
   
    <node name="person_follower_rgb_node" pkg="person_follower_rgb" 
            type="person_follower_rgb_node" required = "true">   

        <param name="target" value="$(arg target)" /> 
        <param name="focal_length" value="$(arg focal_length)" /> 
        <param name="known_target_width_cm" value="$(arg known_target_width_cm)" /> 
        <param name="min_distance" value="$(arg min_distance)" /> 
        <param name="max_distance" value="$(arg max_distance)" /> 
        <param name="speed" value="$(arg speed)" /> 
        <param name="box_percentage_from_image_to_stop" value="$(arg box_percentage_from_image_to_stop)" />

        <param name="angular_scale_factor" value="$(arg angular_scale_factor)" />

        <param name="enable_gestures" value="$(arg enable_gestures)" />

    </node>

    <node name="bbox_convertor_node" pkg="bbox_convertor"
          type="bbox_convertor_node"  output="screen" required = "true">
    </node>	

    <node name="nvidia_to_bbox_convertor_node" pkg="nvidia_skeleton_convertor"
          type="nvidia_to_bbox_convertor_node"  output="screen" required = "true">
    </node>	

</launch>
